POC/ColorPicker/ColorPicker/MainWindow.xaml

```diff
@@ -5,8 +5,43 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:ColorPicker"
         mc:Ignorable="d"
-        Title="MainWindow" Height="450" Width="800">
-    <Grid>
-        
+        Title="One-Click Color" Height="100" Width="400"
+        KeyDown="OnKeyDownHandler">
+    <Grid Margin="6">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="1*"/>
```

Comment:
the 1 is unnecessary.  A plain * is sufficient

Reply:
Noted and changed! Thanks